jsgtk_modules/fs.js

```diff
@@ -134,14 +134,21 @@ const
   }),
   fs = {
     readFile: function readFile(file, options, callback) {
-      // TODO: supports options
-      if (!callback) callback = options;
+      // TODO: support the 'flag' option
+      let encoding = options && options.encoding || options;
+
+      if (!callback) {
+        callback = options;
+        encoding = undefined;
+      }
+
       GFile.new_for_path(file)
         .load_contents_async(null, (source, result) => {
           mainloop.go();
           try {
             let [ok, data, etag] = source.load_contents_finish(result);
             if (!ok) throw 'Unable to read ' + file;
+            if (encoding) data = data .toString(encoding);
```

Comment:
also please remove that space between `data .toString` it just feels wrong 😆 thanks